src/pages/Beta/Stake/PoolCard/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Card, CardHeader, CardStats, CardButtons, TokenName, Label, Value, DetailButton, StakeButton } from './styleds'
+
+export interface PoolCardProps {
+  token: string
+}
+
+const tokens = {
+  Wavax: {
```

Comment:
We're also missing "PNG" from this. So if you look at `master` branch you'll see we should have PNG in here

Reply:
Gotcha. If we merge master to dev, we will have updated configuration, so in staking page, Earn PNG will be showed up automatically